<launch>
    <node pkg="turtle_server" type="turtle_spawn_server.py" name="spawn_turtle_server" output="screen"/>

    <node pkg="turtle_client" type="turtle_spawn_client.py" name="spawn_turtle_client" output="screen">
        <param name="x1" value="$(arg x1)"/>
        <param name="y1" value="$(arg y1)"/>
        <param name="z1" value="$(arg z1)"/>
        <param name="name1" value="$(arg name1)"/>
        <param name="x2" value="$(arg x2)"/>
        <param name="y2" value="$(arg y2)"/>
        <param name="z2" value="$(arg z2)"/>
        <param name="name2" value="$(arg name2)"/>
    </node>

    <node pkg="turtle_client" type="wall.py" name="wall_controller" output="screen"/>
</launch>

